tests/kernel/install_ltp.pm

```diff
@@ -211,10 +211,15 @@ sub add_ltp_repo {
         }
 
         my $arch = '';
-        $arch = "_PowerPC"  if is_ppc64le();
-        $arch = "_zSystems" if is_s390x();
 
-        $arch = ((is_x86_64 || is_aarch64) ? "Tumbleweed" : "Factory") . $arch;
+        # ltp for leap15.2 is available only for x86_64
```

Comment:
The condition below also matches Leap >= 15.3 though

Reply:
Intended, as testing ltp against leap15.{2,3} x86_64 should be enough in current o3 setup. Have I missed leap15.3's kernel testing on other archs ?

Reply:
I don't think kernel testing is enabled for Leap on any arch currently.

Reply:
Yes, currently there is only kernel testing on Tumbleweed, which is tested on all but s390x. I'd either make this working also on Leap 15.3 or explain that in the comment.

Reply:
> Yes, currently there is only kernel testing on Tumbleweed, which is tested on all but s390x. I'd either make this working also on Leap 15.3 or explain that in the comment.

Alright, thought that x86_64 would be enough, I will split the condition for leap15.2 and leap15.3